src/docs/spec-model.md

```diff
@@ -81,3 +81,41 @@ In addition, predicates from the following sources MAY also be encouraged:
 
 * any relation from the [Relation Ontology (RO)](https://obofoundry.org/ontology/ro.html);
 * any relation under [skos:mappingRelation](http://www.w3.org/2004/02/skos/core#mappingRelation) in the [Semantic Mapping Vocabulary](https://mapping-commons.github.io/semantic-mapping-vocabulary/).
+
+## Documenting unmapped elements
+
+The absence of a mapping in a mapping set can be interpreted in two ways:
+
+1. There is no mapping for that specific (absent) mapping that can be established
+2. The mapping was not established for some other reason, for example, the mapping set is a work in progress
+
+To circumvent this ambiguity, we can explicitly state that a specific term cannot be mapped 
+by using a SSSOM-built-in entity called `sssom:NoTermFound`.
+
+Example SSSOM TSV file:
```

Comment:
Leave this kind of example for the user-facing documentation. They do not belong in the spec.